CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+<img src="assets/amber.png" alt="amber logo" width="250" align="right" />
+
+# Contributing to Amber
+This is a simple but exhaustive guide to get you started on contributing to amber.
+
+## Overview
+Amber consists of the following layers:
+
+1. [CLI Interface](#1-cli-interface)
+2. [Compiler](#2-compiler)  
```

Comment:
Maybe also a documentation on how to add a new builtin command?

I was trying to add the `mv` command as example.
And at that point also how to do a new block like `sudo {}`.